include/http/request_handler.js

```diff
@@ -91,6 +91,15 @@ module.exports = function RequestHandlerModule(pb) {
     RequestHandler.staticRoutes = {};
 
     /**
+     * Redirect http codes
+     * @static
+     * @property REDIRECT_CODES
+     * @type {Object}
+     */
+    RequestHandler.REDIRECT_CODES = {MOVED_PERMANENTLY: 301,
```

Comment:
Not something we want to keep up with.  I would use: https://www.npmjs.com/package/http-status-codes